File: rafael/arrays/GeraradorDeRespostasAleatorias.java
class GeraradorDeRespostasAleatorias {
	public static void main(String[] args) {
		int[][] respostas = new int[5][10];
		for(int i = 0; i < respostas.length; i++) {
			for(int j = 0; j < respostas[i].length; j++) {
				respostas[i][j] = (int)(Math.random() * 3 + 1);
				System.out.print(respostas[i][j] + " ");
			}
			System.out.println("aluno " + (i + 1));
		}
	}
}

File: rafael/arrays/GeraradorDeRespostasAleatorias.cs
class GeraradorDeRespostasAleatorias 
{
	static void Main() 
	{
		System.Random gerador = new System.Random();
		int[][] respostas = new int[5][];
		for(int i = 0; i < respostas.Length; i++) 
		{
			respostas[i] = new int[10];
			for(int j = 0; j < respostas[i].Length; j++) 
			{
				respostas[i][j] = (int)(gerador.NextDouble() * 3 + 1);
				System.Console.Write(respostas[i][j] + " ");
			}
			System.Console.WriteLine("aluno " + (i + 1));
		}
	}
}
